### DISCUSSION TITLE
Return to Github from the per-repo search screen

### DISCUSSION THREAD

**Discussion by fgm** [community/community#10824] [Other Feature Feedback, Questions, & Ideas]
When on a per-repo filtered page, there should be an obvious way to see that repo in main github beyond editing the URL

**Comment by colinwm** [comment]
We don't have anything in the UI, but the hotkey `,` will take you to the corresponding page in github.com! 

To see all our hotkeys, you can press `?` anywhere in the app.

**Comment by fgm** [comment]
Thanks. That's a first step, but definitely not a sufficiently accessible / discoverable one.

**Comment by ahmed2m** [comment]
Intuitively I expected clicking on the blue user/repo link would redirect to github.com but I was shocked to find otherwise.
